compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1801,6 +1801,7 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
          else
             return false;
       case TR::vsplats:
+         return true;
```

Comment:
I would prefer listing out the supported data types here since it is still a subset of all `TR::DataType`.